fetch.md

```diff
@@ -1,11 +1,204 @@
+process to a simple prefetch. None of this is set in stone though.
+
+## Fetching With No Credentials
+
+Speculative navigation requests should in no way leak the user's identity to third-party origins. A consequence of
+this is that we must fetch all requests originating from a [prerendering browsing context][4] with no credentials.
+As with the HTTP cache partitioning restrictions, this must apply to all of the following:
+ - Top-level speculative resources
+ - Subresources under top-level speculative resources
+ - Nested browsing contexts and their subresources
+
+Concretely this can be done by overriding the [credentials mode][6] of all requests coming from a
+[prerendering browsing context][4] to the `"omit"` value.
+
+### Credentials After Activation
+
+After a [prerendering browsing context][4] is activated, requests originating from it would no longer have
+their credentials mode overridden, and would be fetched with whatever credentials were present in the user
+agent's cookie store as is typical on the web platform.
+
+> TODO: Discuss the possibility of partitioning the user agent's cookie store. In this proposal, the user
+agent effectively ignores cookies that it already has, for requests initiated from a
+[prerendering browsing context][4]. One downside of this proposal is that new credentials set on resources
+in a [prerendering browsing context][4] will entirely be ignored. If the user agent's cookie store was
+partitioned in a way similar to the HTTP cache, we could use an isolated speculative cookie store for all
+requests in a given [prerendering browsing context][4], and optionally merge cookies in some fashion after
+activation.
+
+## Stripping Referrer Information
+
+To preserve the user's privacy, there should be a limited amount of referrer information sent along with
+speculative navigation requests. Given the sensitive nature of these requests, we believe it should not
+be possible to expose the full referrer path when these requests are cross-origin. Instead, the total referrer
+information will be limited to at most the referrer's origin. If a developer supplies the `referrerpolicy`
+attribute on e.g., a `<link rel=prerender>` it should only be used for tightening the referrer policy beyond
+the platform [default][7] of `strict-origin-when-cross-origin`. We can achieve this by defining a list of
+a list of __sufficiently-strict referrer policies__ that are allowed for these requests.
+
+As mentioned, the list of sufficiently-strict referrer policies are the policies that are as or more
+strict compared to the platform [default][7] of `strict-origin-when-cross-origin`, which includes the
+following policies:
+ - `strict-origin-when-cross-origin`
+ - `same-origin`
+ - `strict-origin`
+ - `no-referrer`
+
+A speculative navigation request supplied with a referrer policy outside of this list will be silently
+cancelled.
```

Comment:
Or not issued in the first place? I'd have expected something like "ignored" rather than "cancelled", which to me suggests that we started it in some way. Disregard if this term is used for consistency with existing language.

Reply:
Yeah, I meant to represent the idea that the prerendering processing model will be aborted early, which really means that the request is ignored, not cancelled. I'll re-word this.